src/models/classes/customer.js

```diff
@@ -9,6 +9,7 @@
     {
         this.name = this.generateName();
         this.order = new CustomerOrder(products);
+        this.timestamp = GAME.model.config.hour;
```

Comment:
Pass this in through the constructor arguments. The model classes should be agnostic of how we present the data to the 'outside' (_i.e._, `Customer` has no direct knowledge of `GAME.model`).